extensions/wikia/CreateNewWiki/js/CreateNewWiki.js

```diff
@@ -275,6 +261,10 @@
 			this.answer = v;
 		},
 
+		onAuthSuccess: function () {
+			window.location.href = decodeURIComponent(this.redirectUrl).replace(/&/, '?');
```

Comment:
I was refreshing it to rerender global nav and show logged in status.

@bkoval replace is needed to correct redirectUrl.